docs/CODE_GUIDELINES.md

```diff
@@ -198,16 +198,18 @@ For example:
 }
 
 /**
- * Full width button
- * 1) Button should take up the full width of the container it lives within
- * 2) Pushes the button away from the other content because it lives in a flexbox container
- * 3) I know this a weird magic number, but [succinct-but-clear explanation]
+ * Full width button.
+ *
+ * Class is doubled to increase specificity.
  */
-.button--full-width {
+.button--full-width.button--full-width {
   background: red;
-  margin-right: auto; /* 2 */
-  width: 100%; /* 1 */
-  height: 37px; /* 3 */
+  /* Pushes the button away from the other content because it lives in a flexbox container */
+  margin-right: auto;
+  /* Button should take up the full width of the container it lives within. */
```

Comment:
Can we remove this comment since it's self explanatory even though it is the docs might be referred to

Reply:
Removed! 👍